install/db/dbmigrate_3.7-3.8.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/***********************************************************
+ Copyright (C) 2020 Siemens AG
+ Author: Gaurav Mishra <mishra.gaurav@siemens.com>
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ ***********************************************************/
+
+/**
+ * @file
+ * @brief Migrate DB from release 3.7.0 to 3.8.0
+ */
+
+use Fossology\Lib\Db\DbManager;
+
+/**
+ * Maximum rows to process at once
+ * @var integer MAX_ROW_SIZE
+ */
+const MAX_ROW_SIZE = 100;
+
+/**
+ * Tables which needs to be recoded
+ * @var array ENCODE_TABLES
+ */
+const ENCODE_TABLES = array(
+  "copyright",
+  "ecc",
+  "keyword"
+);
+
+/**
+ * Calculate the total number of records in concerned tables
+ * @param DbManager $dbManager
+ */
+function calculateNumberOfRecordsInCopyright($dbManager)
```

Comment:
Can you change the naming as the count you get is not specific to only copyright table.

Reply:
But the function is specific to copyright and sister tables.

Do you have any suggestions for it?

Reply:
:D . some thing like calculateNumberOfRecordstablestoencode ?

Reply:
Done, please check.